src/app/silktide-consent-manager.css

```
@@ -0,0 +1,578 @@
+  font-family: var(--fontFamily);
+  color: var(--textColor);
+  font-size: 24px;
+  font-weight: 500;
+  margin: 0px;
+}
+
+#silktide-modal .modal-close {
+  display: inline-flex;
+  border: none;
+  padding: 13px;
+  border: 0px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)